onnx/backend/test/cpp/driver_test.cc

```diff
@@ -20,6 +20,17 @@
 #define ONNXIFI_TESTDATA_EPS 1e-5
 #endif
 
+#define ASSERT_MEMORYSAFE_SUCCESS(X, isGraphInitialized)  \
```

Comment:
i'm not sure what's coding standards for ONNX. I'd favor just a simple method instead of #define (easier write/read/debug/... )

Reply:
People would always argue about things like that :D

http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es31-dont-use-macros-for-constants-or-functions

Reply:
Can we pass X and graph? if you the graph is not initialized, we just pass nullptr, does that sound good to you?

Reply:
@houseroad I guess it is better.